modules/layers/src/scatterplot-layer/scatterplot-layer-fragment.glsl.js

```diff
@@ -21,25 +21,84 @@
 export default `\
 #define SHADER_NAME scatterplot-layer-fragment-shader
 
+#ifdef GL_ES
 precision highp float;
+#endif
+
+#ifdef GL_OES_standard_derivatives
+#extension GL_OES_standard_derivatives : enable
+#endif
 
 varying vec4 vColor;
+varying vec4 bColor;
+
 varying vec2 unitPosition;
 varying float innerUnitRadius;
 
-void main(void) {
+uniform float stroke;
+uniform float outline;
+uniform float strokeWidth;
+
+//Calculation of transition color
+vec4 blendRGBColor(vec4 c1, vec4 c2, float p) {
```

Comment:
Nit: We usually write out the parameter names, it makes the function more self-documenting.